tapchannel/auf_leaf_signer_test.go

```diff
@@ -131,6 +142,171 @@ func TestAuxLeafSigner(t *testing.T) {
 	}
 }
 
+// TestAuxLeafSignerCancel tests that the AuxLeafSigner will handle a cancel
```

Comment:
If I run this vanilla in master, I get this error: 
```
 FAIL: TestAuxLeafSignerCancel (0.05s)
=== RUN   TestAuxLeafSignerCancelAndQuit
panic: close of closed channel

goroutine 5 [running]:
github.com/lightninglabs/taproot-assets/tapchannel.(*AuxLeafSigner).processAuxSigBatch.func1(...)
        /Users/roasbeef/gocode/src/github.com/lightninglabs/taproot-assets/tapchannel/aux_leaf_signer.go:250
github.com/lightninglabs/taproot-assets/tapchannel.(*AuxLeafSigner).processAuxSigBatch(0x140006c7080, 0x140001c6008, 0x140012a9220, {0x14000748008, 0xa, 0x1039c0840?})
        /Users/roasbeef/gocode/src/github.com/lightninglabs/taproot-assets/tapchannel/aux_leaf_signer.go:259 +0x588
created by github.com/lightninglabs/taproot-assets/tapchannel.(*AuxLeafSigner).SubmitSecondLevelSigBatch in goroutine 4
        /Users/roasbeef/gocode/src/github.com/lightninglabs/taproot-assets/tapchannel/aux_leaf_signer.go:104 +0xc8
exit status 2
```

Reply:
Yes, that's the expected outcome - added the tests separately from the fix to show the problem if lnd hit an error and closed the channel.